torchinductor/codegen/wrapper.py

```diff
@@ -97,6 +98,8 @@ def __init__(self):
             ["", "", f"def call({', '.join(V.graph.graph_inputs.keys())}):"]
         )
         with self.prefix.indent():
+            for name in V.graph.randomness_seeds:
+                self.prefix.writeline(f"{name}.fill_(random.randrange(2**31))")
```

Comment:
Is it better to get a seed from `random.randrange` or from torch cpu-side generator? I'm not sure if typical call to `torch.manual_seed` also sets a seed for python-side random generation, so determinism could be lost?
Also, this won't work with cuda graphs, because this argument to `fill_` kernel will be baked in, and will be used on all subsequent calls. Handling of random + cuda graphs on the eager side is a bit delicate, with the state being advanced at each replay, and read in the kernels.

Reply:
I changed this to use `torch.randint`